tests/common.bash

```diff
@@ -497,11 +497,19 @@ function ensure_yq() {
 
 # dependency: What we want to get the version from the versions.yaml file
 function get_from_kata_deps() {
-        local dependency="$1"
         versions_file="${repo_root_dir}/versions.yaml"
 
         command -v yq &>/dev/null || die 'yq command is not in your $PATH'
-        result=$("yq" read -X "$versions_file" "$dependency")
+
+        yq_version=$(yq --version | grep -oE '[0-9]+\.[0-9]+\.[0-9]+' | cut -d. -f1)
+        if [ "$yq_version" -eq 3 ]; then
+          dependency=$(echo "$1" | sed "s/^\.//g")
+          result=$("yq" read "$versions_file" "$dependency")
+        else
+          dependency=$1
+          result=$("yq" "$dependency | explode (.)" "$versions_file")
+        fi
```

Comment:
Do you think we need the support for v3 & v4 here?

Reply:
I don't remember why I added this, but probably because some tests were failing. This whole yq is in need of some love, and in some places is still calling from older version, depending on the architecture. I would much like to keep as this and remove later in a follow up pr.